cdap-ui/app/directives/widget-container/widget-input-schema/widget-input-schema.html

```diff
@@ -52,6 +53,12 @@
           errors="errors[inputschema.name]">
         </my-complex-schema>
       </div>
+      <div
```

Comment:
Should the comments at the beginning of this file be updated to reflect the changes?